website/src/components/Chat/ChatListBase.tsx

```diff
@@ -15,11 +15,24 @@ import { HEADER_HEIGHT } from "../Header/Header";
 import { ChatListItem } from "./ChatListItem";
 
 export const ChatListBase = memo(function ChatListBase(props: CardProps) {
-  const { data: chats } = useSWR<GetChatsResponse>(API_ROUTES.LIST_CHAT, get, {
+  const { data: chats, mutate: mutateChats } = useSWR<GetChatsResponse>(API_ROUTES.LIST_CHAT, get, {
     revalidateOnFocus: true,
   });
   const { t } = useTranslation(["common", "chat"]);
 
+  const handleUpdateTitle = useCallback(
+    ({ chatId, title }: { chatId: string; title: string }) => {
+      mutateChats(
+        (chatResponse) => ({
+          ...chatResponse,
+          chats: chatResponse?.chats.map((chat) => (chat.id === chatId ? { ...chat, title } : chat)) || [],
+        }),
+        false
+      );
```

Comment:
is this necessary? I assume we update the title in the `ChatItem`, and then we just refresh here? do we need to re-assign the title here again?.

Reply:
Just for instantly update. It also saves us a request since we already have enough information to display. The second param `false` means not to revalidate.

Reply:
aha, I see, I assumed if we wanted to show instant updates, we would do that in the `ChatItem` directly? maybe it is simpler?

Reply:
How can we update it in the ChatItem? Using a state?

Reply:
would be the most straight forward solution. what do you think?

Reply:
I see, totally forgot about the other menu for mobile.